src/utils.js

```diff
@@ -16,5 +18,12 @@ export const getWikipediaAttrFromUrl = url => {
 
 		return null
 	},
+
 	isTouch = 'ontouchstart' in window || ( navigator.maxTouchPoints > 0 ) ||
-		( navigator.msMaxTouchPoints > 0 )
+		( navigator.msMaxTouchPoints > 0 ),
+
+	addMiniGalleryRow = ( lang, title ) => {
+		requestPageMedia( lang, title, mediaData => {
+			renderPreviewMedia( document, mediaData )
```

Comment:
missing case, when there is no mediaData (empty array), we don't need to render the gallery.